prow.sh

```diff
@@ -1237,6 +1333,22 @@ function version_gt() {
     greaterVersion=${greaterVersion#"v"};
     test "$(printf '%s' "$versions" | sort -V | head -n 1)" != "$greaterVersion"
 }
+# shellcheck disable=SC2120
+install_yq_if_missing() {
+  local version="${1:-v4.48.1}"
+  local yq_path="/usr/local/bin/yq"
+  if ! command -v yq &>/dev/null; then
+    echo "Installing yq ${version}..."
+    if ! curl -fsSL -o "${yq_path}" "https://github.com/mikefarah/yq/releases/download/${version}/yq_linux_amd64"; then
+      echo "Failed to download yq" >&2
+      exit 1
+    fi
+    chmod +x "${yq_path}"
+    echo "yq ${version} installed at ${yq_path}"
+  else
+    echo "yq found, skipping install"
+  fi
+}
```

Comment:
lets install this in `${CSI_PROW_BIN}/yq` path rather than trying to write to `/usr/local/bin` which may not be writable always.